buildout.cfg

```diff
@@ -1,5 +1,5 @@
 [buildout]
-extends = https://raw.githubusercontent.com/zopefoundation/Zope/master/versions.cfg
+extends = http://zopefoundation.github.io/Zope/releases/master/versions.cfg
```

Comment:
This should be https. (There is a redirect from http to https though, so it probably works.)

Reply:
Good catch!